components/alertbanner/index.css

```diff
@@ -92,17 +101,18 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-AlertBanner-icon {
-  inline-size: var(--mod-alert-banner-icon-size, var(--spectrum-alert-banner-icon-size));
-  block-size: var(--mod-alert-banner-icon-size, var(--spectrum-alert-banner-icon-size));
-  margin-block-start: var(--mod-alert-banner-top-icon, var(--spectrum-alert-banner-top-icon));
+  --mod-icon-size: var(--mod-alert-banner-icon-size, var(--spectrum-alert-banner-icon-size));
```

Comment:
This mod passthrough for icon size is here rather than on `.spectrum-AlertBanner` so that it doesn't affect Button in the case it also uses an icon (or if there was an icon used in the content).